style.css

```diff
@@ -87,34 +87,34 @@ body.very-hazardous {
   color: #FFF;
 }
 
-body .explanation {
+body .maybe-show {
 	display: none;
 }
 
-body.very-hazardous .explanation.very-hazardous {
-	display: block;
+body.very-hazardous .maybe-show.very-hazardous {
+	display: revert;
```

Comment:
Woah, I never knew about `revert`!

Reply:
I didn't either! While I wasn't paying any attention, display: has grown a host of new possible values. BITD, it was none, block, inline or the oh-so-fancy inline-block.

For anyone reading this PR later: the previous behavior made all of our maybe-shows into block elements, which isn't what we wanted for spans that are supposed to show up inline in headings.